src/collection/account.service.ts

```diff
@@ -388,7 +388,7 @@ export class AccountService {
             serviceName,
             httpErr,
           );
-          throw httpErr;
+          //throw httpErr;
```

Comment:
Caller should check if empty or valid GitHub repo before calling and still throw here. Maybe it should have an env config?